x-pack/solutions/search/plugins/search_indices/public/index_management_application.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreSetup, ScopedHistory } from '@kbn/core/public';
+import { I18nProvider } from '@kbn/i18n-react';
+import { IndexManagementPluginSetup } from '@kbn/index-management-shared-types';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { IndexManagementApp } from './components/index_mangement/app';
+
+export const renderIndexManagementApp = async (
```

Comment:
I suspect this file isn't needed as this functionality is already provided within the index management app.

Reply:
Thank you for suggestion, I am working on this change and will update PR if all looks good.

Reply:
I think this is still needed, especially the `IndexManagementApp` rendered here allows us to wrap the IndexManagement component in a page template that will render the search side nav when in classic solution nav.

Reply:
+1 @TattdCodeMonkey's comment,  based on my testing we need to have providers in [IndexManagementApp](https://github.com/elastic/kibana/blob/069e3811278f0cf8a44125ef7d74b0da6f594bf1/x-pack/solutions/search/plugins/search_indices/public/components/index_mangement/app.tsx#L24) to render classic solution nav. 
Although we could remove some redundant providers in renderIndexManagementApp , I think we will still need to have `KibanaContextProvider` to provide `history` 

Updated in this [commit](https://github.com/elastic/kibana/pull/209599/commits/70bf5e95ecd0dd344a2f19d67831ce0b9ccdce3d)

Reply:
The general exercise is for the interface for both usages to be the same, similar to the params arg. 

>Although we could remove some redundant providers

Yes, we definitely want that to happen, or ideally define all the providers in a single place.

Reply:
Okay, looks like `KibanaContextProvider` is used by your own code, rather than anything Index Management needs.

Reply:
@mattkime  Also added `KibanaRenderContextProvider` & `I18nProvider` in this [commit](https://github.com/elastic/kibana/pull/209599/commits/7341afc746bebb20e91b6288f30008866c536039).  Without these providers, side nav is rendered empty when navigated to index management list page from classic nav( path: Elasticsearch -> Content-> Index management)